kong/cmd/utils/migrations.lua

```diff
@@ -145,6 +134,7 @@ local function up(schema_state, db, opts)
 
       assert(db:run_migrations(schema_state.executed_migrations, {
         run_up = true,
+        suppress_teardown = true,
```

Comment:
Why is `force` only executing the `up` state of migrations? Since executed migrations have previously ran both steps (`up` and `teardown`), saying that `force` "forces re-execution of these migrations" is incorrect. It will only force re-execution of the `up` step of these migrations, but the user may need to re-execute the `teardown` phase instead? I really do not like this `--force` option.

Reply:
Do you mean `suppress_teardown`, Or do you mean that it only passes `run_up=true`. Do you mean that I should change what it originally does?

Reply:
Yes, I can hear that it is confusing. I guess it could be because the `teardowns` are not as re-entrant as `ups`.

Reply:
Though on the other hand, it can be simply wrong to not execute `teardown` too.

Reply:
It was implemented here: https://github.com/Kong/kong/pull/4071

Reply:
I have now changed `migrations up -f` and also added `migrations finish -f`